### PR TITLE
Hotfix to login and register styles

### PR DESCRIPTION
**Pull request by adrianMRJ**
# Description
 - What?: Fix styles on login and register.
 - Why?: Because they were not acording to the design.

## Screenshots

 - Mobile
image
image

 - Desktop
image
image

### CODE REVIEW

File: stylesheets/abstracts/_variables.scss [hunk 1]
```diff
@@ -3,3 +3,5 @@ $dark-grey: #464343;
 $grey: #E5E5E5;
 $white: #FFFFFF;
 $background-home: #F6F6F6;
+$input-grey: #EBEAEA;
+$input-border: #D9D9D9;
```
Comment (JuanequeX) [inline stylesheets/abstracts/_variables.scss:7]: Add blank space here

File: stylesheets/views/_login.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      background-color: #F6F6F6;
+      border-radius: 1.5rem;
+      padding-top: 2rem;
+      margin-top: 1rem;
+      margin-bottom: 1rem;
+      width: 20rem;
+    }
+  }
+
+
```
Comment (JuanequeX) [inline stylesheets/views/_login.scss:29]: Delete this extra space

File: stylesheets/views/_register.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  
+    &__logo {
+      margin-left: 10rem;
+    }
+  
+    &__header {
+      font-weight: 700;
+      margin-top: 1.5rem;
+    }
+
```
Comment (JuanequeX) [inline stylesheets/views/_register.scss:37]: Delete this extra space

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: stylesheets/abstracts/_variables.scss (hunk 1 above)
Comment (adrianMRJ) [reply stylesheets/abstracts/_variables.scss:7]: Done

File: stylesheets/views/_register.scss (hunk 3 above)
Comment (adrianMRJ) [reply stylesheets/views/_register.scss:37]: Done

File: stylesheets/views/_login.scss (hunk 2 above)
Comment (adrianMRJ) [reply stylesheets/views/_login.scss:29]: Done

**Review by JuanequeX — DISMISSED** [review]
LGTM

**Review by fabiannabi — DISMISSED** [review]
LGTM

**Review by JuanequeX — APPROVED** [review]
LGTM

**Review by EdsonMorfin — APPROVED** [review]
lgtm

**Pull request merged**